atm.js

```diff
@@ -1,21 +1,84 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var chkDeposit = $("#checking_deposit");
+var chkWithdrawal = $("#checking_withdraw");
+var savDeposit = $("#savings_deposit");
+var savWithdrawal = $("#savings_withdraw");
+var chkAmount = 0
+var savAmount = 0
 
-}
 
-function checkingWithdrawal(amount){
+// an eventListener for each button, each one a "click"
+chkDeposit.on("click", checkingDeposit);
+chkWithdrawal.on("click", checkingWithdrawal);
+savDeposit.on("click", savingsDeposit);
+savWithdrawal.on("click", savingsWithdrawal);
 
-}
+//actions responding to each button
+function checkingDeposit() {
+  chkAmount = parseInt($("#checking_amount").val());
+  var checkingBal = $(".balance.checking").html();
+  var chkBalAmt = parseInt(checkingBal.replace("$", ""));
+  checkingBal = chkBalAmt + chkAmount;
+  $(".balance.checking").text("$" + checkingBal);
+  if(checkingBal == 0) {
+    $(".balance.checking").addClass("zero");
+  } else {
+    $(".balance.checking").removeClass("zero");
+  };
+};
 
-function savingsDeposit(amount){
 
+function checkingWithdrawal(){
+  chkAmount = parseInt($("#checking_amount").val());
+  var checkingBal = $(".balance.checking").html();
+  var chkBalAmt = parseInt(checkingBal.replace("$", ""));
+  var savingsBal = $(".balance.savings").html();
```

Comment:
Use variable names when selecting elements with jQuery to increase your app's performance.